tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py37
+envlist = py311
```

Comment:
can you change this to py38, py39, py310, and py311? would be nice to see if we can run on all versions.